doc/users/next_whats_new/logit_scale_stuff.rst

```diff
@@ -0,0 +1,13 @@
+Improvements in Logit scale ticker and formatter
+-----------------------------------------------
```

Comment:
```
/home/circleci/project/doc/users/next_whats_new/logit_scale_stuff.rst:2: WARNING: Title underline too short.

Improvements in Logit scale ticker and formatter
-----------------------------------------------
```

Reply:
I blame my spell checker , which correct the title without correction the underline.

Will be fixed.

Reply:
fixed, committed, ans squashed.